### ISSUE TITLE
Tin and copper not all stackable

### ISSUE THREAD

**Issue by mushroomhostage** [mushroomhostage/exphc#1]
Sometimes you mine copper ore, and mine some more ore, and it creates two different stacks There are two different types of copper ore and copper ingots. Same problem with tin.

**Comment by mushroomhostage** [comment]
The problem is both IndustrialCraft2 and RedPower2 generate their own copper and tin ores - Forestry too, but I disabled its tin/copper generation (now only generates apatite). This is not the most serious problem, since both types can be user interchangeably (thanks to the Forge ore dictionary), but it is annoying.

Fixed by disabling RP2's generation, in redpower/redpower.cfg:

```
world {
    generate {
        copper=0
        emerald=1
        indigo=1
        marble=1
        nikolite=1
        rubbertree=1
        ruby=1
        sapphire=1
        silver=1
        tin=0
        volcano=1
    }
```

Unfortunately, 30% of the world was already generated with the duplicate ores.

**Comment by mushroomhostage** [comment]
One option: regenerate the whole map, with the new settings – not that appealing, takes overnight, and everything will be wiped.

Another possibility: https://code.google.com/p/midas-gold/ - change existing blocks in map?

Or change drop? http://dev.bukkit.org/server-mods/otherdrops/

**Comment by mushroomhostage** [comment]
https://github.com/mushroomhostage/exphc/wiki/Data-values

Copper: IC=249, RP=140;5
Tin: IC=248, RP=140;4

**Comment by mushroomhostage** [comment]
Trying with this plugins/OtherDrops/otherdrops-config.yml:

```
# RedPower copper drops IndustrialCraft copper
X140@5:
 - drop: X249
# RedPower tin drops IndustrialCraft tin
X140@4:
 - drop: X248
```

but no such luck. 140:5 still drops 140:5. Maybe be easier to write a simple plugin to change this drop instead of figuring out OtherDrops.

**Comment by mushroomhostage** [comment]
https://github.com/mushroomhostage/ExphcTweaks/commit/cd52bb984dbdc679386fd19ccacb34c8b84a6f1b Drop IC2 tin/copper when RP2 tin/copper ore is mined

**Comment by mushroomhostage** [comment]
Deployed on exphc, works fine. There is still the problem of existing mined ores and ingots from each mod not stacking, but at least new ores will be consistent.

**Comment by mushroomhostage** [comment]
Finally fixed properly in 1.2.3 but disabling tin and copper ore generation in RedPower2 and Forestry, let IC^2 handle it.
